torch/_ops.py

```diff
@@ -59,9 +59,6 @@ def __repr__(self):
     def __call__(self, *args, **kwargs):
         return self._op(*args, **kwargs or {})
 
-    def __getattr__(self, key):
```

Comment:
why does this work?  is key also in the attrs of self, or are we now causing an AttributeError and that's desirable for some reason?

Reply:
I removed this mostly because I didn't know why we have it, and because in a subsequent PR I am going to redefine getattr for other reasons

Reply:
ok.